src/main/controller/UserController.ts

```diff
@@ -83,7 +84,7 @@ export class UserController {
           UserRepository
         );
         const result = await userRepository.save(user);
-        return res.status(201).send({ ...result, password: undefined });
+        return res.status(201).send({ ...result, idFirebase: undefined });
```

Comment:
Just curious but is there a reason why we're sending back `idFirebase` if we'll be setting it to undefined?

Reply:
just like password, BE do not have to send `uid` to FE since FE should get `uid` from current user of firebase.

Reply:
Oooh could we just remove idFirebase completely?

Reply:
it just removes idFirebase from response data. I think there's no huge effect on our app even if the response data contains `idFirebase`. However, I want to just make sure that we can get `uid` when we signed up/in correctly. Every time FE requests some data to BE, the current user would be supposed to be authorized. So, when it creates new user on our DB, the user in FE can get `uid` by itself that means BE does not have to contain `idFirebase` in the response data.